Lesson_2_JSON_YAML_XML/house.xml

```diff
@@ -0,0 +1,37 @@
+<house>
+    <rooms>
+        <kitchen>
+            <table>1</table>
+            <chairs>5</chairs>
+            <oven>1</oven>
+            <cleaned>true</cleaned>
+            <state>redecorated</state>
+        </kitchen>
+        <living_room>
+            <table>0</table>
+            <chairs>2</chairs>
+            <TV_set>1</TV_set>
+            <cleaned>false</cleaned>
+            <state>shabby</state>
+        </living_room>
+        <bedroom>
+            <table>0</table>
+            <chairs>0</chairs>
+            <bed>1</bed>
+            <TV_set>1</TV_set>
+            <cleaned>true</cleaned>
+            <state>shabby</state>
+        </bedroom>
+    </rooms>
+    <owners>
+        <husband>
```

Comment:
Overall it's not a mistake, but makes the automated processing of array items harder:

If we say the array contains consistent items -> `<owners>` child elements can be just identified as `<owner>` elements
The information about who's a wife and who's a husband can be added as a property of an `<owner>` element like

```
<owners>
    <owner>
        <name>Amy</name>
        <age>34</age>
        ....
        <maritalTitle>wife</maritalTitle>
    </owner>
</owners>
```

In this case, some program that will have to deal with this object (while parsing it) -> will recognize <owners> as a collection/array/list of similar objects -> which is always easier to work with

Reply:
changed, thanks